fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/service/InternalProgressiveLoanApiResourceSwagger.java

```diff
@@ -37,7 +37,7 @@ private ProgressiveLoanInterestScheduleModel() {}
         @Schema(example = "[]")
         public TreeSet<InterestRate> interestRates;
         @Schema(example = "{}")
-        public Map<Integer, List<LoanTermVariationsData>> loanTermVariations;
+        public Map<Integer, List<InternalLoanTermVariationsData>> loanTermVariations;
```

Comment:
I am not fond of this new name.... Why is this needed?

Reply:
ohh.. it is internal api... my bad... then it is fine... i guess

Reply:
Or we can remove the serialization from the ApiResource and delete this whole swagger file :D

Reply:
I spoke with @somasorosdpc and he said that I could delete the swagger file as well as any integration tests that depend on it because we have similar tests elsewhere.